playwright/README.md

```diff
@@ -9,23 +9,28 @@
 
```

Comment:
In line 8 above, "Playwright" should have an upper case "P". I know it was there already, but I thought that as the file is being amended anyway then it could be corrected :)